apps/dotcom/client/src/tla/hooks/useAppState.tsx

```diff
@@ -49,21 +54,33 @@ export function AppStateProvider({ children }: { children: ReactNode }) {
 			return
 		}
 		let _app: TldrawApp
+		let _userId: RecordId<TldrawAppUser>
+
+		const userPreferencesRaw = getUserPreferences()
+		const userPreferences = atom<TLUserPreferences>('userPreferences', userPreferencesRaw)
```

Comment:
we shouldn't be using this built-in user preferences or creating an atom here, we should be creating something which stores prefs on the `TldrawAppUser` record in the store. This could be an ordinary property on the TldrawApp class.

so we'd end up doing something like

```ts
export classs TldrawApp {
  ...
  tlUser = createTLUser({
    userPreferences: computed('user prefs', () => {
       const user = this.getUser(this.getCurrentUserId())
       return {
         ... extract prefs from user record
       }
    }),
    setUserPreferences: (prefs) => {
       this.store.put([{
          ... update the current user
       }])
    }
  })
}
```

looks like you added a property for this on TldrawAppUser but forgot to actually use it?